youtube_dl/extractor/atvat.py

```diff
@@ -28,7 +28,7 @@ def _real_extract(self, url):
         display_id = self._match_id(url)
         webpage = self._download_webpage(url, display_id)
         video_data = self._parse_json(unescapeHTML(self._search_regex(
-            r'class="[^"]*jsb_video/FlashPlayer[^"]*"[^>]+data-jsb="([^"]+)"',
+            r'var flashPlayerOptions[ ]*=[ ]*"([^\n]+)";',
```

Comment:
1. Do not remove the old regex.
2. `\s*` for matching whitespace.
3. When matching between quotes - match any non quote character.
4. Remove useless `;`.

Reply:
fixed